### ISSUE TITLE
SynLIDAR validation set

### ISSUE THREAD

**Issue by andrewcaunes** [saltoricristiano/cosmix-uda#20]
Hello ,

Thank you for your great work !
I noticed in the paper you mention about SynLIDAR (Sec 4.1) "We follow the authors’ instructions [32], and use 19,840 point clouds for training and 1,976 point clouds for validation."

However, after searching in the SynLIDAR paper, repo, and other papers from the authors, I can't find an official train/val split anywhere. Also, after downloading the 13 sequences of the SubDataset, I only found 19,840 scans and not the additionnal "1,976 point clouds for validation.".

Could you clarify what validation set you used please ? This would be helpful for better reproduction accuracy ! 

Thanks in advance,
Andrew

**Comment by lichonger2** [comment]
There are two versions of the Synlidar dataset, please download the FULL version.

**Comment by andrewcaunes** [comment]
Hello, I also downloaded the full version but the CosMIX paper specifically indicates following the SynLIDAR authors' instructions to use the SubDataset.
Also even in the full dataset, there are no specification for a train/val split. The FullDataset is just the original version with denser scans of the SubDataset, they follow the same sequences.

**Comment by lichonger2** [comment]
The authors used the full version of the Synlidar dataset and selected 19,840 of these point clouds as the training set and an additional 1,976 as the validation set. The two sets of point clouds do not overlap.
Please read **synlidar.py**. The code in there clearly shows the logic of this part.Thanks!

**Comment by andrewcaunes** [comment]
Oh, ok, thank you for the synlidar.py reference.
So the authors randomly sampled 19,840 scans for training from the FullDataset, instead of using the pre-samples 19,840 scans from SubDataset, and additionnaly sampled 1,976 scans for validation.

I think I will stick with the provided SubDataset for better replication, and use one of the sequences for validation since taking a scan from the same sequences is not great for validating generalization capability.

Thanks for the help,
Andrew
